lib/framework.js

```diff
@@ -1173,15 +1183,17 @@ Framework.prototype.onMessageCreated = function (message) {
             }
           }
 
-          _.forEach(matched, lex => {
+          _.forEach(matched, (lex, i) => {
```

Comment:
Note, this is not part of the new command/prompt functionality.  This is the bug fix for membership rules sending multiple messages when multiple hears() phrases match.  By passing the index of the match into the `shouldCallHears()` function, that function can send the message only on the first match for any given message.